plaid/item.go

```diff
@@ -102,8 +102,21 @@ type CreatePublicTokenResponse struct {
 }
 
 type createItemAddTokenRequest struct {
-	ClientID string `json:"client_id"`
-	Secret   string `json:"secret"`
+	ClientID   string                  `json:"client_id"`
+	Secret     string                  `json:"secret"`
+	UserFields *ItemAddTokenUserFields `json:"user,omitempty"`
+}
+
+// VerificationDateUnknown indicates that the email/phone was verified, but the
+// time of verification is not known.
+var VerificationDateUnknown = time.Unix(0,0)
```

Comment:
is this significantly different from a nil emailAddressVerifiedAt?

Reply:
Yes, it allows us to distinguish three cases: 
1. unverified fields (missing date)
2. verification at unknown date (sentinel value)
3. verification at known date (any other date value).

The "date as bool" is not a great pattern but it's what we've already - painstakingly - agreed in various design committees 😅